pkg/services/auth/token_cleanup.go

```diff
@@ -24,7 +24,7 @@ func (s *UserAuthTokenService) Run(ctx context.Context) error {
 	for {
 		select {
 		case <-ticker.C:
-			err = s.ServerLockService.LockAndExecute(ctx, "cleanup expired auth tokens", time.Hour*12, func() {
+			err = s.ServerLockService.LockAndExecute(ctx, "cleanup expired auth tokens", time.Hour*12, func(context.Context) {
```

Comment:
Now I wonder, if you are using the ctx inside the closure anyway - does it make sense to pass context as a parameter if it remains unused? Would there be a case when these two contexts are different?

Reply:
https://github.com/grafana/grafana-enterprise/pull/1983/files this is why I added it